source/Services/xdr-agent-crime.html.md.erb

```diff
@@ -0,0 +1,189 @@
+---
+title: XDR Agent Installation - Crime
+last_reviewed_on: 2024-11-19
+review_in: 12 months
+weight: 156
+---
+
+# <%= current_page.data.title %>
+
+We have developed a method of installing the XDR agent to Crime VMs. This differs from the HMCTS install method which utilised the VM Extension capability on Azure VMs and is delivered via Terraform.
+
+For Crime, we are utilising Ansible as the delivery method and have developed an Ansible Role which will perform the installation.
+
+To trigger the Ansible, we have created two pipelines in the main Crime Jenkins instance which can be scope limited by Ansible's Limit functionality. This enables us to follow a rollout plan through environments and safely stage our delivery.
+
+When the deploy pipeline is run, this calls an Ansible playbook that includes a Role we have developed which handles the installation.
+
+This document details the procedure to do this.
+
+
+
+## Ansible Inventory/Variables
+
+Before running the pipelines, two variables need declaring in either Ansible group_vars or host_vars. These are:
+
+- cortex_env
+- xdr_tags
+
+cortex_env is a string which needs set to either 'nonprod' or 'prod'. MoJ SoC provided a separate installation package for nonprod & prod, this defines which installation package is used.
+
+xdr_tags is a comma-separated string of tags to provide to the Cortex Agent.
+
+Example:
+
+```bash
+xdr_tags: "hmcts,server,idam"
+cortex_env: "nonprod"
+```
+
+An MS Team channel exists called "HMCTS - Tagging Catch Up" with the MoJ SoC team as members. Please reach out for help if unsure on tags on VMs.
```

Comment:
Would maybe just change this to reach out to MoJ SOC if unsure of tags to use - bit more generic for the future?